src/Event/Event.php

```diff
@@ -123,7 +123,13 @@ public function getAvailableRoles(): array
 
     public function getLogoInBase64(): string
     {
-        return ImageUtils::getLocalImageInBase64($this->logoUrl);
+        return ImageUtils::getLocalImageInBase64(self::getFullLogoUrl($this->logoUrl));
+    }
+
+    public static function getFullLogoUrl(?string $logoUrl): string
+    {
+        if ($logoUrl === null) { return "";}
+        return "/assets" . $logoUrl;
```

Comment:
Takto natvrdo vypálená adresa do kódu není dobře - když to budeme chtít přesunout, tak budeme muset změnit kód, což je špatně. Lepší by bylo toto zanést do infra, aby webserver hledal v této složce automaticky a kód appky to vůbec nemusel vědět